### PR TITLE
Update genesis for mainnet launch

### PR DESCRIPTION
**Pull request by zjshen14**
1. Time stamp and eth height is changed to 4/21, and estimated height around 4/21's 7pm pdt.
2. Exempt 36 robots from epoch bonus
3. Allow 200M token in the reward fund
4. Make only one address to lock 9800M token (@raullenchai , you need to provide an address here)

### CODE REVIEW

File: genesis_mainnet.yaml [hunk 1]
```diff
@@ -1,29 +1,6 @@
...
-    io1scs89jur7qklzh5vfrmha3c40u8yajjx6kvzg9: "100000000000000000000000000"
-    io1u5ff879gg2dw9vfpxr2tsmuaz07e2rea6gvl7s: "100000000000000000000000000"
-    io1v0q5g2f8z6l3v25krl677chdx7g5pwt9kvqfpc: "100000000000000000000000000"
-    io1vtm2zgn830pn6auc2cvnchgwdaefa9gr4z0s86: "100000000000000000000000000"
-    io1wv5m0xyermvr2n0wjx2cjsqwyk863drdl5qfyn: "100000000000000000000000000"
-    io1x9kjkr0qv2fa7j4t2as8lrj223xxsqt4tl7xp7: "100000000000000000000000000"
-    io1xsdegzr2hdj5sv5ad4nr0yfgpsd98e40u6svem: "100000000000000000000000000"
-    io1xsx5n94kg2zv64r7tm8vyz9mh86amfak9ka9xx: "100000000000000000000000000"
-    io1z7mjef7w528nasnsafan0rp6yuvkvq405l6r8j: "100000000000000000000000000"
+    io12yxdwewry70gr9fs6fphyfaky9c7gurmzk8f4f: "9800000000000000000000000000" # Need to update the address
```
Comment (raullenchai) [inline genesis_mainnet.yaml]: pls use io1uqhmnttmv0pg8prugxxn7d8ex9angrvfjfthxa

File: genesis_mainnet.yaml [hunk 2]
```diff
@@ -72,9 +54,15 @@ rewarding:
...
+    - io10kyvvzu08pjeylymq4umknjal25ea3ptfknrpf
+    - io18mvepyxkcd5jkyplfqn27ydkpsendrey3xe2l8
+    - io1nz40npqa3yvek4zdasmqaetl2j4h6urejfkera
+    - io1m7p9yrejngxyvxhvn7p9g9uwlvd7uuamg8wcjd
+    - io1cwwm08dwv9phh3wt5vsdhu9gcypw9q2sc7pl9s
+    - io14aj46jjmtt83vts9syhrs9st80czumg40cjasl
   foundationBonus: "80000000000000000000"
   foundationBonusLastEpoch: 8760
-  initBalance: "1200000000000000000000000000"
+  initBalance: "200000000000000000000000000"
```
Comment (raullenchai) [inline genesis_mainnet.yaml]: can we later send more IOTX to the rewarding pool? is it tested?

Comment (zjshen14) [reply genesis_mainnet.yaml]: now gas is sent to the pool, but we don't have the cli to do it now

Comment (zjshen14) [reply genesis_mainnet.yaml]: Will add and verify

**Pull request merged**